### PR TITLE
refactor: 💡 Refactored rates API

### PR DESCRIPTION
**Pull request by kostysh**
The `rates` API is refactored to `/rate/{facilityId}/{itemId}/{key}/{rateType}`

### CODE REVIEW

File: test/api.spec.ts [hunk 1]
```diff
@@ -411,7 +411,7 @@ describe('API tests', async () => {
     describe('Rates', async () => {
       it('should be throw err if rate not exist', async () => {
         await requestWithSupertest
-          .get(`/api/rate/${facilityId}/items/${spaceId}/default`)
+          .get(`/api/rate/${facilityId}/${spaceId}/default`)
```
Comment (Galeaf11) [inline test/api.spec.ts]: /rate/{facilityId}/{itemId}/{key}/{rateType}:
in swagger 
it's works?

Comment (Galeaf11) [reply test/api.spec.ts]: it should be `/api/rate/${facilityId}/${spaceId}/default/items`

Comment (kostysh) [reply test/api.spec.ts]: you are right, `items` must be moved to the end

**Review by Galeaf11 — APPROVED** [review]
LGTM

**Pull request merged**